src/app/pages/CreateConnectorPage/DuplicateConnectorPage.tsx

```diff
@@ -0,0 +1,150 @@
+import { getConnector, getConnectorTypeDetail } from '@apis/api';
+import { CreateConnectorWizard } from '@app/components/CreateConnectorWizard/CreateConnectorWizard';
+import { CreateConnectorWizardProvider } from '@app/components/CreateConnectorWizard/CreateConnectorWizardContext';
+import { Loading } from '@app/components/Loading/Loading';
+import { useCos } from '@context/CosContext';
+import { fetchConfigurator } from '@utils/loadFederatedConfigurator';
+import React, {
+  FunctionComponent,
+  useState,
+  useEffect,
+  useCallback,
+} from 'react';
+import { useTranslation } from 'react-i18next';
+import { useLocation } from 'react-router-dom';
+
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  Button,
+  Level,
+  Modal,
+  PageSection,
+  Title,
+} from '@patternfly/react-core';
+
+import { useBasename, useConfig } from '@rhoas/app-services-ui-shared';
+import { Connector, ConnectorType } from '@rhoas/connector-management-sdk';
+
+type DuplicateConnectorPageProps = {
+  onSave: () => void;
+  onClose: () => void;
+};
+export const DuplicateConnectorPage: FunctionComponent<DuplicateConnectorPageProps> =
+  ({ onSave, onClose }) => {
+    const { t } = useTranslation();
+    const config = useConfig();
+    const basename = useBasename();
+    const { connectorsApiBasePath, getToken } = useCos();
+    const [askForLeaveConfirm, setAskForLeaveConfirm] = useState(false);
+    const openLeaveConfirm = () => setAskForLeaveConfirm(true);
+    const closeLeaveConfirm = () => setAskForLeaveConfirm(false);
+
+    const [connectorData, setConnectorData] = useState<Connector>();
+    const { hash } = useLocation();
+    const connectorId = hash.split('&')[0].substring(1);
+    const [duplicateMode, setDuplicateMode] = useState<boolean | undefined>();
+    const getConnectorData = useCallback((data) => {
+      setConnectorData(data as Connector);
+    }, []);
+
+    const [connectorTypeDetails, setConnectorTypeDetails] =
+      useState<ConnectorType>();
+
+    const onError = useCallback((error) => {
+      console.log(error);
+    }, []);
+
+    const getConnectorTypeInfo = useCallback((data) => {
+      setConnectorTypeDetails(data as ConnectorType);
+    }, []);
+
+    useEffect(() => {
+      if (connectorId !== undefined) {
+        setDuplicateMode(true);
+      }
+      // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
What does disabling this lint rule in this file fix?

Reply:
@gashcrumb It is to avoid exhaustive dependencies to pass.